src/ledger/test/LedgerCloseMetaStreamTests.cpp

```diff
@@ -427,3 +431,68 @@ TEST_CASE("METADATA_DEBUG_LEDGERS works", "[metadebug]")
     }
     REQUIRE(gotToExpectedSize);
 }
+
+TEST_CASE("meta stream contains reasonable meta", "[ledgerclosemeta]")
+{
+
+    TmpDirManager tdm(std::string("metatest-") + binToHex(randomBytes(8)));
+    TmpDir td = tdm.tmpDir("meta-ok");
+    std::string path = td.getName() + "/stream.xdr";
+
+    VirtualClock clock;
+    Config cfg = getTestConfig();
+    cfg.METADATA_OUTPUT_STREAM = path;
+
+    {
+        // Do some stuff
+        using namespace stellar::txtest;
+        auto app = createTestApplication(clock, cfg);
+        auto bal = app->getLedgerManager().getLastMinBalance(2);
+
+        auto root = TestAccount::createRoot(*app);
+
+        // Ledger 2 creates the account, which we do directly so emit no meta
+        // for.
+        auto acc1 = root.create("acc1", bal);
+
+        // Ledger 3 uses closeLedger so emits interesting meta.
+        std::vector<TransactionFrameBasePtr> txs = {
```

Comment:
I would add a couple other transactions in there (as the bug we had was that meta was broken with ledgers with more than 1 tx).
For good measure one of these could be a fee bump transaction

Reply:
done